docs/source/guide/export.md

```diff
@@ -1,33 +1,103 @@
 ---
-title: Export results
+title: Export results from Label Studio
 type: guide
-order: 105
+order: 402
 ---
 
-Your annotations are stored in [raw completion format](#Completion-format) inside `my_project_name/completions` directory, one file per labeled task named as `task_id.json`.
+## Export data from Label Studio
 
-You can optionally convert and export raw completions to a more common format by doing one of the following:
+Export your completed annotations from Label Studio. Label Studio stores your annotations in a raw JSON format in the SQLite database backend or whichever cloud or database storage you specify as target storage, with one file per labeled task named as `task_id.json`. 
```

Comment:
one file per labeled task is only relevant to export cloud storages, not databases